client/src/i18n/en-US/index.js

```diff
@@ -315,6 +315,12 @@ export default {
     style_criteria: {
       heading: "Style Criteria",
       edit_button: "Edit",
+      delete_confirm: "Are you sure you want to delete {name}?",
+      delete_header: "Confirm Delete Criteria",
+      addBtnLabel: "Add Criteria",
+      saveError:
+        "Oops, there was an error saving, check the form above for errors.",
+      deleteError: "Oops, unable to delete.",
```

Comment:
I'm not a fan of "Oops" being used in error messages but I won't let my reservation prevent this PR from proceeding.

Reply:
I've no concern with your making an issue to change them.